packages/central-server/src/apiV2/getCountryAccessList.js

```diff
@@ -18,29 +18,34 @@ const mapRequestsToEntities = (requestedEntities, projectCodeById) => {
 };
 
 export async function getCountryAccessList(req, res, next) {
-  const { userId, models } = req;
+  const { userId, models, query, accessPolicy } = req;
+  const { projectCode } = query;
+  if (!projectCode) throw new Error('No project code provided');
 
   try {
-    const countries = await models.entity.find({ type: 'country' });
-    const entityPermissions = await models.userEntityPermission.find({ user_id: userId });
-    const permittedEntityIds = new Set(entityPermissions.map(p => p.entity_id));
-    const accessRequests = await models.accessRequest.find({
-      user_id: userId,
-      processed_date: null,
-    });
-    const projects = await models.project.find();
-    const projectCodeById = reduceToDictionary(projects, 'id', 'code');
-    const entityRequests = mapRequestsToEntities(accessRequests, projectCodeById);
+    const project = await models.project.findOne({ code: projectCode });
+    if (!project) throw new Error(`No project found with code ‘${projectCode}’`);
```

Comment:
IDE complain about this throw in a `try` block (not unreasonable, I suppose). Are we concerned?